.github/workflows/e2e.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         uses: submariner-io/shipyard/gh-actions/e2e@devel
         with:
           using: ${{ matrix.cable_driver }} ${{ matrix.deploytool }} ${{ matrix.globalnet }} ${{ matrix.ovn }}
+          focus: '\[.*\]'
+          skip: '\[external_dataplane\]'
```

Comment:
This can be safely be defined here.
However, I meant to make it available [here](https://github.com/submariner-io/shipyard/blob/devel/gh-actions/e2e/action.yaml#L26) in shipyard side, but forgot to change from '-' to '_'. (Ginkgo's regrex asked to change, so.)

Should we fix it in shipyard first and remove this or do we keep this and just fix it in shipyard later?

Reply:
Forget this. '\[external-dataplane\]' seems to be a valid focus and skip expression. Just applied to E2E test and github workflows.